src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.task.Task;
+
+/**
+ * A TaskCard Controller class
+ */
+public class TaskCard extends UiPart<Region> {
```

Comment:
This is a good abstraction. Much needed to visualise the tasks for now. Will need to revisit this when changing the GUI in the near future.